configure.d/1_submit_bio.conf

```diff
@@ -14,18 +14,21 @@
 check() {
 	cur_name=$(basename $2)
 	config_file_path=$1
-	if compile_module $cur_name "submit_bio(NULL);" "linux/bio.h"
+	if compile_module $cur_name "void *t=submit_bio(NULL);" "linux/bio.h"
```

Comment:
submit bio typically returns int status. wouldn't this cause compilation error here when casted to void *?

Reply:
Also the return value is never used. Let's just make cas_submit_bio() always have no return value and not differentiate between the two variants of submit_bio (the one returning void and the one returning blk_qc_t)